pkg/lib/cockpit-po-plugin.js

```diff
@@ -80,7 +81,7 @@ module.exports = class {
 
         Array.prototype.push.apply(patterns, extras);
 
-        return patterns.map((p) => new RegExp(`^${p}:[0-9]+$`));
+        return patterns.map((p) => new RegExp(`^${this.filePrefix}${p}:[0-9]+$`));
```

Comment:
I guess we should be escaping the path to guard against regexp-y characters that we'd find in there, no?

That was probably already true for the `${p}` (and this would be a welcome cleanup) but now that we include other paths, we might imagine seeing some more 'exotic' things there...

Reply:
We might also just check that the string has the expected prefix, to be honest.  I'm not entirely 100% sure why I did it like this...